lib/build-view.js

```diff
@@ -17,6 +17,7 @@ module.exports = (function() {
     this.titleLoopIndex = 0;
     this.a2h = new Convert();
     this.monocle = false;
+    this.attach();
```

Comment:
Ah. I see.

You are correct. I think this is the best way too, and it works well. Thanks!